src/main/scala/ai/eto/rikai/sql/model/ModelSpec.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2021 Rikai authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai.eto.rikai.sql.model
+
+import scala.collection.JavaConverters.mapAsJavaMap
+
+/** Model Spec is used to pass the create model information to
+  * python ModelRegistry.
+  */
+class ModelSpec(
+    val name: Option[String],
+    val uri: String,
```

Comment:
nit: let's call this spec_uri as well to be consistent with the new python code?

Reply:
sure